plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1060,8 +1069,11 @@ else if (strings.size() != 2) {
             return viewToken;
         }
 
-        String storageTableName = Optional.ofNullable(materializedViewDefinition.get().getStorageTable()).orElse("");
```

Comment:
was it a bug?

Reply:
no, a change is needed here due to change in storageTable field in ConnectorMaterializedViewDefinition

Reply:
`.orElse("");` seems problematic since `getIcebergTable` was called next